### PR TITLE
Merge in upstream changes from Jico's repo

### PR DESCRIPTION
**Pull request by lydiastepanek**
Referenced https://github.com/lodash/lodash/issues/1852

### CODE REVIEW

File: lib/enhance.js [hunk 1]
```diff
@@ -6,10 +6,13 @@
```
Comment (lydiastepanek) [inline lib/enhance.js]: Will this cause any issues?

File: lib/lodash-ext.js [hunk 2]
```diff
@@ -2,15 +2,27 @@
 (function() {
   var _;
 
-  _ = require('../vendor/lodash');
+  _ = require('../vendor/lodash/dist/lodash');
+  _.noConflict();
```
Comment (lydiastepanek) [inline lib/lodash-ext.js]: Added this (diff suppressed for some reason)

File: src/lib/enhance.coffee [hunk 3]
```diff
@@ -21,10 +24,16 @@ Enhance = do ->
       if window.matchMedia?(query).matches then return true
       return false
 
+    isMobileDevice = ->
+      mobileDevice = if options.tabletAsMobile then 'tablet' else 'phone'
+      breakpoint   = options["#{mobileDevice}Breakpoint"]
+      window.matchMedia?("only screen and (max-width: #{breakpoint}px)").matches
+
+
```
Comment (kevinc) [inline src/lib/enhance.coffee]: extra line

File: src/lib/enhance.coffee [hunk 4]
```diff
@@ -36,7 +45,9 @@ Enhance = do ->
```
Comment (kevinc) [inline src/lib/enhance.coffee]: extra line

**Comment by armyofgnomes** [comment]
Does this upgrade the lodash version too or is it just Jico's changes?

**Comment by lydiastepanek** [comment]
@armyofgnomes No, but will do in next pull. Game plan is:
1) Merge in upstream changes (this pull)
2) Production acceptance by deploying a feature branch in header that uses this version of enhance
3) Update lodash verion
4) repeat #2 
5) See if Sentry errors calm down
6) If not, add in lodash noConflict()

**Comment by armyofgnomes** [comment]
lgtm

**Comment by kevinc** [comment]
:+1:

**Pull request closed without merging**